### ISSUE TITLE
Where is the function assigned to Office.initialize()?

### ISSUE THREAD

**Issue by user135711** [OfficeDev/office-js-docs-pr#708]
The document states: 
"so leave the function that is assigned to Office.initialize with an empty body. (But do not delete it. The Office.initialize property cannot be null or undefined.)"

But there is no assignment of or in the IIFE: 
(function () {
  
    Office.onReady()
    .then(function() {

        //If you need to initialize something you can do so here. 

    });
})();

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ac7d9e2d-b537-cf1f-95fb-4e216514f3b1
* Version Independent ID: 4a86c93b-70d8-38e4-f62e-83c43a0158dd
* Content: [Excel add-in tutorial - Office Add-ins](https://docs.microsoft.com/en-us/office/dev/add-ins/tutorials/excel-tutorial)
* Content Source: [docs/tutorials/excel-tutorial.md](https://github.com/OfficeDev/office-js-docs-pr/blob/master/docs/tutorials/excel-tutorial.md)
* Product: **office 365**
* GitHub Login: @o365devx
* Microsoft Alias: **o365devx**

**Comment by user135711** [comment]
Also, it might be useful to add a statement that these add-ins don't work unless you are connected to the internet. Maybe give a reason why as I'm interested to know.

**Comment by user135711** [comment]
that's what I would've thought, but even before the reaching the part of the tutorial where I saw scripts included from web addresses, the add-in only worked on my VM when it had access to the internet. It would be an incredible coincidence otherwise.

**Comment by Rick-Kirkham** [comment]
You're correct. The tutorial loads the Office.js from a CDN. I'll add a note. Thanks.

**Comment by Rick-Kirkham** [comment]
The fixed version should be online sometime tomorrow night.
